tests/optics/focus-lens.test.tsx

```diff
@@ -11,10 +11,10 @@ const succ = (input: number) => input + 1
 
 it('basic derivation using focus works', async () => {
   const bigAtom = atom({ a: 0 })
-  const propA = (optic: O.OpticFor<{ a: number }>) => optic.prop('a')
+  const focusFunction = (optic: O.OpticFor<{ a: number }>) => optic.prop('a')
```

Comment:
Would you please make changes for other `tests/optics/*.test.tsx` files?

Reply:
aight done :)